.github/workflows/stale.yml

```diff
@@ -0,0 +1,35 @@
+# https://docs.github.com/en/actions
+
+name: "Issues and pull requests"
+
+on:
+  schedule:
+    - cron: "0 12 * * *"
+
+env:
+  DAYS_BEFORE_CLOSE: 90
+  DAYS_BEFORE_STALE: 1095
+
+jobs:
+  prune:
+    name: "Issues"
+
+    runs-on: "ubuntu-latest"
+
+    steps:
+      - name: "Prune stale issues and pull requests"
+        uses: "actions/stale@v5"
+        with:
+          days-before-close: "${{ env.DAYS_BEFORE_CLOSE }}"
+          days-before-stale: "${{ env.DAYS_BEFORE_STALE }}"
+          repo-token: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
is it sth that is there by default, or we need to create it in project settings?

Reply:
@keradus 

This secret is provided automatically!